src/DynamoCore/Models/DynamoModelCommands.cs

```diff
@@ -315,6 +319,35 @@ void BeginConnection(Guid nodeId, int portIndex, PortType portType)
             }
         }
 
+        void BeginDuplicateConnection(Guid nodeId, int portIndex, PortType portType)
+        {
+            // If the port clicked is an output port, begin connection as per normal
+            if (portType == PortType.Output)
+            {
+                BeginConnection(nodeId, portIndex, portType);
+                return;
+            }
+
+            // Otherwise, if the port is an input port, check if the port already has a connection.
+            // If it does, duplicate the existing connection.
+            var node = CurrentWorkspace.GetModelInternal(nodeId) as NodeModel;
+            if (node == null)
+            {
+                return;
+            }
+
+            PortModel portModel = node.InPorts[portIndex];
```

Comment:
var

Reply:
Is this a required standard?  I don't understand why it's bad practice to be explicit when you know the type for readability.

Reply:
this is a good guide:
https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/inside-a-program/coding-conventions#implicitly-typed-local-variables

I am pretty confident this type is a PortModel because I know the Inports type is always going to be a portModel list.